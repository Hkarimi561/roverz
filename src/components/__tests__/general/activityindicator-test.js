/**
 * Test to check if the component renders correctly
 */
/* global it expect */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Preloader from '@components/general/ActivityIndicator';

it('Preloader renders correctly', () => {
  const tree = renderer.create(
    <Preloader />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
