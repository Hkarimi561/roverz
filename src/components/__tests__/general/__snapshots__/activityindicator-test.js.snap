// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader renders correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="rgba(255,255,255,0.3)"
  hidesWhenStopped={true}
  size="large"
  style={
    Array [
      Array [],
    ]
  }
/>
`;
